pkg/agent/qrm-plugins/cpu/dynamicpolicy/state/state.go

```diff
@@ -231,13 +238,54 @@ func (ai *AllocationInfo) CheckSideCar() bool {
 	return ai.ContainerType == pluginapi.ContainerType_SIDECAR.String()
 }
 
+func (ai *AllocationInfo) GetPodAggregatedRequest() (float64, bool) {
+	if ai.Annotations == nil {
+		return 0, false
+	}
+	value, ok := ai.Annotations[apiconsts.PodAnnotationAggregatedRequestsKey]
+	if !ok {
+		// TODO remove this code after upgrade existed kubelet
+		value, ok = ai.Annotations[consts.PodDeprecatedAnnotationAggregatedRequestsKey]
+		if !ok {
+			return 0, false
+		}
+	}
+	var resourceList v1.ResourceList
+	if err := json.Unmarshal([]byte(value), &resourceList); err != nil {
+		general.Errorf("failed to unmarshal pod aggregated request list: %q", err)
+		return 0, false
+	}
+
+	return float64(resourceList.Cpu().MilliValue()) / 1000, true
+}
+
+func CheckAllocationInfoTopologyAwareAssignments(ai1, ai2 *AllocationInfo) bool {
```

Comment:
move CheckAllocationInfoTopologyAwareAssignments, CheckAllocationInfoOriginTopologyAwareAssignments,  checkCPUSetMap to state/util.go